jwst/nsclean/nsclean.py

```diff
@@ -0,0 +1,387 @@
+import logging
+import numpy as np
+
+from astropy.stats import sigma_clipped_stats
+
+from jwst import datamodels
+from .. assign_wcs import nirspec
+from stdatamodels.jwst.datamodels import dqflags
+
+from jwst.nsclean.lib import NSClean, NSCleanSubarray
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def mask_ifu_slices(input_model, mask):
+
+    """Find pixels located within IFU slices and flag them in the
+    mask, so that they do not get used.
+
+    Parameters
+    ----------
+    input_model : data model object
+        science data
+
+    mask : 2D boolean array
+        input mask that will be updated
+
+    Returns
+    -------
+    mask : 2D boolean array
+        output mask with additional flags for science pixels
+    """
+
+    import gwcs
```

Comment:
Why is this import here? Move it to the top of the module?

Reply:
moved it